index.js

```diff
@@ -1,12 +1,20 @@
 'use strict';
 
 const AWS = require('aws-sdk');
-const proxy = require('proxy-agent')
+const proxy = require('proxy-agent');
 
 module.exports = {
   initialize: (conf) => {
-    let awsConf = {}
-      , configurationKeys = ['accessKeyId', 'secretAccessKey', 'region', 'proxy', 'maxRetries', 'retryDelayOptions'];
+    const awsConf = {};
+    const configurationKeys = [
+      'accessKeyId',
+      'secretAccessKey',
+      'region',
+      'proxy',
+      'maxRetries',
+      'retryDelayOptions',
+      'visibilityTimeout'
```

Comment:
Yeah we might want to do that or move this under a different type of config.  These are all options to the top-level AWS config (http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html) but it looks like visibilityTimeout is something we're just using in the `AWS.SQS` calls?